tool/lib/bert-annotator.py

```diff
@@ -34,16 +34,32 @@ def template_query(cat, query_canonical, prefix, value='', suffix=''):
     :param suffix: the suffix of the canonical form
     :return: a template query string
     """
+    question_start = "who" if query_canonical == "person" else f"which {query_canonical}"
     if cat == 'base':
-        return f"what is the {prefix} of the {query_canonical} ?".split()
+        return [
+            f"what is the {prefix} of the {query_canonical} ?".split(),
+            f"what is the {query_canonical} 's {prefix} ?".split(),
+            f"what {prefix} does the {query_canonical} have ? ".split(),
+        ]
     if cat == 'property':
-        return f"show me a {query_canonical} with {prefix} {value} {suffix} .".split()
+        return [
+            f"show me a {query_canonical} with {prefix} {value} {suffix} .".split(),
+            f"{question_start} has {prefix} {value} {suffix} ?".split()
+        ]
     if cat == 'verb':
-        return f"which {query_canonical} {prefix} {value} {suffix} ?".split()
+        return [
+            f"{question_start} {prefix} {value} {suffix} ?".split(),
+            f"show me a {query_canonical} that {prefix} {value} {suffix} .".split()
+        ]
     if cat == 'passive_verb':
-        return f"show me a {query_canonical} {prefix} {value} {suffix} .".split()
+        return [
+            f"show me a {query_canonical} {prefix} {value} {suffix} .".split(),
+            f"{question_start} is {prefix} {value} {suffix} .".split()
+        ]
     if cat == 'reverse_property':
-        return f"which {query_canonical} is a {prefix} {value} {suffix} ?".split()
+        return [
+            f"{question_start} is a {prefix} {value} {suffix} ?".split()
+        ]
     raise Exception('Invalid grammar category: ', cat)
```

Comment:
I feel like we should move this somewhere else, maybe some data file, to make it easy to extend in the future (for other languages, for other use cases, for other types of annotations)